docs/argo-server-sso.md

```diff
@@ -112,13 +112,24 @@ If no rule matches, we deny the user access.
 
 It is possible to have the Argo Workflows Server use the Argo CD Dex instance for SSO, for instance if you use Okta with SAML which cannot integrate with Argo Workflows directly. In order to make this happen, you will need the following:
 
-- You must be using at least Dex [v2.23.0](https://github.com/dexidp/dex/releases/tag/v2.23.0), because that's when `staticClients[].secretEnv` was added.
-- A secret created above with a `client-id` and `client-secret` to be used by both Dex and Argo Workflows Server. It is called `argo-workflows-sso` in this example.
+- You must be using at least Dex [v2.23.0](https://github.com/dexidp/dex/releases/tag/v2.23.0), because that's when `staticClients[].secretEnv` was added. That means ArgoCD 1.7.12 and above.
+- A secret containing two keys, `client-id` and `client-secret` to be used by both Dex and Argo Workflows Server. `client-id` is `argo-workflows-sso` in this example, `client-secret` can be any random string. If ArgoCD and ArgoWorkflows are installed in different namespaces the secret must be present in both of them. Example:
+  ```yaml
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: argo-workflows-sso
+  data:
+    # client-id is 'argo-workflows-sso'
+    client-id: YXJnby13b3JrZmxvd3Mtc3Nv
+    # client-secret is 'MY-SECRET-STRING-CAN-BE-UUID'
+    client-secret: TVktU0VDUkVULVNUUklORy1DQU4tQkUtVVVJRA==
+  ```
 - `--auth-mode=sso` server argument added
 - A Dex `staticClients` configured for `argo-workflows-sso`
 - The `sso` configuration filled out in Argo Workflows Server to match
 
-What this might look like in your chart configuration:
+### Example Helm chart configuration for authenticating against ArgoCD's Dex
```

Comment:
can we put this in its own document please?

Reply:
What do you mean, which part? One document for Helm, another for the manifests? Or the entire SSO-via-ArgoCD-Dex section in its own document?

Reply:
one document for using Argo DEX

Reply:
How does it look now?